### ISSUE TITLE
Add data layers to "imagery used" in changeset tags

### ISSUE THREAD

**Issue by SomeoneElseOSM** [openstreetmap/iD#4984]
Currently, iD users don't add sources to changesets, but iD adds details of the imagery they used as a tag.  It'd be great if it also included the "data layer overlays" in that too.

For example, here's someone changing access to a road based on a traffic sign seen in a mapillary photo:
https://www.openstreetmap.org/changeset/57960469 

and a lot of the initial conversation was about "where did you see that?", which could have been avoided if the overlay layer used could have been included somehow.

The date of the photo would also have been really useful here but I guess that might be tricky with many photo dates...

**Comment by JamesKingdom** [comment]
@SomeoneElseOSM I tried this [here](https://master.apis.dev.openstreetmap.org/changeset/112862), and it displays `Esri World Imagery;Mapillary Images;Mapillary Signs` in the `imagery_used` box.

`counter_striker` is still on iD 2.4.1, and is hiding the `host` box (I think this was discussed somewhere).
I think they are changing their `imagery_used` tag, as the feature to display that Mapillary had been used was added in commit https://github.com/openstreetmap/iD/commit/a8ac9faad8a597d81813d8effd5cc5c1a1a0a668 (2016) , which is iD version 1.9.6.
